tests/integration/modules/test_ssh.py

```diff
@@ -132,7 +132,9 @@ def test_recv_known_host_entries(self):
         """
         Check that known host information is returned from remote host
         """
-        ret = self.run_function("ssh.recv_known_host_entries", ["github.com"])
+        ret = self.run_function(
+            "ssh.recv_known_host_entries", ["github.com"], **dict(enc="ssh-rsa")
```

Comment:
**question** any reason to use dict unpacking here, instead of just `enc="ssh-rsa"`?

Reply:
Thanks, fixed, unpacking was just the first thing which came into my mind